toolkit/pythonit_toolkit/api/service_client.py

```diff
@@ -0,0 +1,40 @@
+from typing import Any, Dict, Optional
+
+import httpx
+from pythonit_toolkit.headers import SERVICE_JWT_HEADER
+from pythonit_toolkit.pastaporto.tokens import generate_token
+
+
+class ServiceClient:
+    def __init__(
+        self,
+        url: str,
+        issuer: str,
+        audience: str,
+        jwt_secret: str,
+    ):
+        self.url = url
+        self.issuer = issuer
+        self.audience = audience
+        self.jwt_secret = jwt_secret
+
+    async def execute(
+        self,
+        document: str,
+        variables: Optional[Dict[str, Any]] = None,
+    ):
+        token = generate_token(
+            self.jwt_secret, issuer=self.issuer, audience=self.audience
+        )
+
+        async with httpx.AsyncClient() as client:
+            response = await client.post(
+                self.url,
+                data={"query": document, "variables": variables},
+                headers={
+                    SERVICE_JWT_HEADER: token,
+                },
+            )
+
+            data = await response.json()
+            return data
```

Comment:
maybe we can parse this and return a dataclass

```
class ServiceResponse:
    errors: Optional[List]
    data: Optional[str]
```

so in the future we could play a lot more with it and do stuff like `.has_errors` and so on